### DISCUSSION TITLE
When minio cluster upgrade from the version 2022-03-26T06 to the lasted, downloading for a long time error with connection refused happens

### DISCUSSION THREAD

**Discussion by suntao1983** [minio/minio#16420] [Q&A]
When minio cluster upgrade from the version 2022-03-26T06 to the lasted, downloading for a long time error with connection refused happens：
Error: Marking minio-2:9000 offline temporarily: caused by Post "http://minio-2:9000/minio/storage/dataset/minio6/v49/readall?disk-id=&file-path=format.json&volume=.minio.sys: dial tcp10.77.115.3:9000: connect: conection refused"

**Comment by vadmeste** [comment]
@suntao1983 is minio-2 crashing sometimes ?

**Comment by suntao1983** [comment]
No, it works . but it really happens when i download a big file(9GB) maybe 90% downloaded,  the error happened in minio-1 when loadFormatEnrasure ,while minio-2 happened another error: unexcepted EOF when  nsscanner. I guess is there anything change between the 2 versions  in scanner flow?
